src/PlaywrightSharp.Firefox/FirefoxRequest.cs

```diff
@@ -16,7 +17,7 @@ public FirefoxRequest(FirefoxSession session, Frame frame, List<Request> redirec
             Id = payload.RequestId;
```

Comment:
Can we rename this guy to `FirefoxInterceptableRequest`? so we are closer to pw.